src/test/integration/IntegrationBase.t.sol

```diff
@@ -499,6 +641,31 @@ abstract contract IntegrationBase is IntegrationDeployer {
         return curShares;
     }
 
+    /// @dev Uses timewarp modifier to get staker shares at the last snapshot
```

Comment:
Line 631 above: https://github.com/Layr-Labs/eigenlayer-contracts/blob/36203255b143550de64b7e093a55a48f4e9a2fcc/src/test/integration/IntegrationBase.t.sol#L626-L633

Should we now remove this revert message? Perhaps we just use ints instead of uints across the board when asserting shares? - `assert_Snap_Unchanged/Added/Removed_StrategyShares` could just pass in zero/positive/negative share deltas into `assert_Snap_Delta_StakerShares`

Reply:
Actually DM only uses uint256...

Reply:
I have no idea... I want to avoid having to do a ton of casting on the test level

Reply:
If the casting is under the hood I think that'd be fine

Reply:
ya it's fine for now